back/engines/commercial/report_builder/app/services/report_builder/queries/analytics/base.rb

```diff
@@ -41,4 +41,10 @@ def project_filter(project_id_column, project_id)
   def interval(resolution)
     RESOLUTION_TO_INTERVAL.fetch(resolution || 'month')
   end
+
+  def visitor_filter(apply)
+    return {} unless apply
+
+    { 'dimension_user.is_visitor': 'true' }
```

Comment:
Is this a thing? I could not find an `is_visitor` column in the `dimension_users` table.

Reply:
Aaah, you added it in the other PR. There might be an issue with the new definition of the view though. The LEFT JOIN will duplicate users that are associated to multiple visits, I think.